tower/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+# Unreleased
+
+### Added
+- Added "full" feature which turns on all other features.
```

Comment:
nit/TIOLI: I might mark this as "Fixed", since the docs in previous releases mentioned this existing, but it was missing?

Reply:
Ah right. I didn't realize it used to exist. Fixed in 3f4127a9e901a.